bigbluebutton-html5/package.json

```diff
@@ -62,6 +65,7 @@
     "postcss-modules-scope": "1.1.0",
     "postcss-modules-values": "1.3.0",
     "postcss-nested": "2.0.2",
+    "selenium-webdriver": "3.0.0-beta-2",
```

Comment:
beta packages?

Reply:
All later versions work only with Node 6.x+, but Meteor is still using 4.x.
We can try to update it once Meteor updates its Node version (which is supposed to happen in Meteor 16).

Reply:
Meteor 16 brings in Node 8.x